### ISSUE TITLE
Including a Post or Page from within a Template

### ISSUE THREAD

**Issue by awolfe76** [jekyll/jekyll#873]
Is it possible to include a post or a page within a template. 
The posts/pages are not in, and would not be in, the _includes directory.

For example I want a template that will act as the home page for a section of a site. I would like it to get the content for 3 of the pages in the site and output the desc of those pages.

something like {% include content1.md %}, which currently doesn't work of course if the the content.md file isn't in the _includes directory.

Basically I just want to be able to include content (pages/posts), other than the page/post being currently generated, from within a template.

Thanks,
Andrew
